router.go

```diff
@@ -598,6 +598,11 @@ func (r *Router) GetRoute(name string) *Route {
 	return r.namedRoutes[name]
 }
 
+// GetRoutes get name of routes.
+func (r *Router) GetRoutes() map[string]*Route {
```

Comment:
:) 按golang语法该叫 `NamedRoutes()` 😄 

```suggestion
func (r *Router) NamedRoutes() map[string]*Route {
```